src/style.css

```diff
@@ -1,3 +1,96 @@
+:root {
+  --blue-color: rgba(0, 172, 220, 1);
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+  font-family: Roboto, serif;
+  font-weight: 400;
+}
+
+header {
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+}
```

Comment:
Tag name 'header' is used for styling. It's better to use class selectors instead of tag selectors for styling, except for 'html' and 'body'. Please consider adding a class to the 'header' element and use that for styling.